languages/ruby/exercises/concept/README.md

```diff
@@ -6,12 +6,13 @@ The concept exercises are based on this [list of concepts][reference-shared].
 
 These are the concept exercises that have currently been implemented, as well as the concepts they teach and their prerequisite concepts:
 
-| exercise                                                            | concepts                             | prerequisites                |
-| ------------------------------------------------------------------- | ------------------------------------ | ---------------------------- |
-| [`strings`][concept-exercise-strings]                               | `strings`                            | -                            |
-| [`basics`][concept-exercise-basics]                                 | `basics`                             | -                            |
-| [`numbers`][concept-exercise-numbers]                               | `numbers`<br/>`conditionals`         | `basics`                     |  |
-| [`floating-point-numbers`][concept-exercise-floating-point-numbers] | `floating-point-numbers`<br/>`loops` | `numbers`<br/>`conditionals` |
+| exercise                                                            | concepts                             | prerequisites                                           |
+| ------------------------------------------------------------------- | ------------------------------------ | ------------------------------------------------------- |
+| [`strings`][concept-exercise-strings]                               | `strings`                            | -                                                       |
+| [`basics`][concept-exercise-basics]                                 | `basics`                             | -                                                       |
+| [`numbers`][concept-exercise-numbers]                               | `numbers`<br/>`conditionals`         | `basics`                                                |
+| [`floating-point-numbers`][concept-exercise-floating-point-numbers] | `floating-point-numbers`<br/>`loops` | `numbers`<br/>`conditionals`                            |
+| [`arrays`][concept-exercise-arrays]                                 | `arrays`<br/>`each-loops`            | `classes`<br/>`chars`<br/>`booleans`<br/>`conditionals` |
```

Comment:
This line has `each-loops` as a concept that is unlocked (that was removed) and `chars` as a prerequisite (which should be `blocks`).